<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorPreferences</class>
 <widget class="QWidget" name="EditorPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>270</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QGroupBox" name="fontGroup">
     <property name="title">
      <string>&amp;Font</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="fontLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>F&amp;amily:</string>
          </property>
          <property name="buddy">
           <cstring>fontSelect</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentFont">
           <font>
            <family>DejaVu Sans Mono</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="sizeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Size:</string>
          </property>
          <property name="buddy">
           <cstring>sizeSpin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sizeSpin"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="indentGroup">
     <property name="title">
      <string>&amp;Indentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="indentLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Tab &amp;width:</string>
          </property>
          <property name="buddy">
           <cstring>indentSpin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="indentSpin">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="noTabsBox">
        <property name="text">
         <string>I&amp;nsert spaces instead of tabs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="indentBox">
        <property name="text">
         <string>Enable &amp;auto-indentation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fontSelect</tabstop>
  <tabstop>sizeSpin</tabstop>
  <tabstop>indentSpin</tabstop>
  <tabstop>noTabsBox</tabstop>
  <tabstop>indentBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
